.github/ISSUE_TEMPLATE.md

```diff
@@ -1,13 +1,21 @@
+Please, check https://docs.docker.com/docker-for-win/troubleshoot/.
+Issues without logs and details cannot be exploited, and will be closed.
```

Comment:
exploited? probably "debugged" or "diagnosed" is better

Reply:
Thanks for catching this...  This is a Frenchism :(
> ▸ exploiter une situation
> to capitalize on a situation

Let's go for debugged.